metaseq/data/streaming_src_tgt_dataset.py

```diff
@@ -48,6 +49,7 @@ def __init__(
         self.drop_last = drop_last
         self.padding_idx = padding_idx
         self.shuffle_buffer_size = shuffle_buffer_size
+        self.left_truncation = left_truncation
```

Comment:
Nit: renaming this to something like "--target-first-truncation"? I think three months later I will forget what "left_truncation" means lol

Reply:
Actually, "target-first-truncation" is also confusing. Since we deal with left-right LMs, its implied that target is on the end of the block and the current name: `left_truncation` seems appropriate to me for *this* dataset module.